frontend/src/about.html

```diff
@@ -1,100 +1,93 @@
-      </p>
-      <p>
-        Once you start playing any audio, the playback continues in the background while you can continue to filter and search for other audios. Soon we will add a floating audio player with most
-        features. Currently each file can be downloaded individually, but going forward we will be adding a bulk download option. Also, brief topics for each lecture, transcripts, and many more
-        usability features will be added in the upcoming months.
-      </p>
-      <p>
-        Although we have tried our best, we are sure there will still be ways to improve the audios and we look forward to your inputs - whether it is proposing a better title, or looking into audio
-        content issues. To enable this, the website has a feature where feedback can be provided for each audio specifically. Use a
-        <span class="whitespace-nowrap">
-          Feedback button
-          <svg class="inline-block h-auto w-4 stroke-current" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
-            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M8 10h.01M12 10h.01M16 10h.01M9 16H5a2 2 0 01-2-2V6a2 2 0 012-2h14a2 2 0 012 2v8a2 2 0 01-2 2h-5l-5 5v-5z" />
-          </svg>
-        </span>
-        in the search result item for this. You can also give general feedback on the website using the 'Contact Us' menu item. Feel free to report any issue or propose an improvement or a new
-        functionality.
-      </p>
-      <h4>How we worked with the AUDIO FILES:</h4>
-      <p>
-        Although in the later years of Srila Gurudeva's preaching, lectures were officially recorded in an organised manner, this archive, to a great extent, has been aggregated from personal cassette
-        recordings of various devotees. We had to work with the raw material we had. You may notice several inconsistencies in the recordings - Some devotees only recorded when Srila Gurudeva spoke
-        and stopped recording when other speakers, called upon to speak by Srila Gurudeva, spoke. Some devotees had good quality recorders which auto-changed the cassette side so that none of the
-        recording was lost while changing sides. But in many cases some katha was lost while changing the cassette side. Thus we have a lot of cases of abrupt beginnings and endings. Some devotees
-        recorded in bits and pieces on different cassettes. This made it very difficult for us to piece together those lectures, especially if some of those cassettes were missing or corrupt. If low
-        quality cassettes were used, then the voice frequencies are impaired, some are warped and distorted. In such cases we just saved whatever we could.
-      </p>
-      <p>To bring some method to the overall exercise, we followed these rules while working on these files:</p>
+---
+layout: base
+title: Srila BV Narayana Gosvami Maharaj’s Official Archive
```

Comment:
This is the generic website title, it shouldn't be specified here. Here should be the "About" title.

Reply:
changed.